### ISSUE TITLE
bug [Financial times no longer works]

### ISSUE THREAD

**Issue by ashprice** [bpc-clone/bypass-paywalls-firefox-clean#22]
Describe issue/request: Financial times has started to show the subscription page again. 

Link of paywalled article: Any FT.com article, such as https://www.ft.com/content/d64b1fc1-59b6-4fbd-8a88-8d786e4df474

Actions you tried yourself and if it works (clear cookies, disabe Javascript, change user-agent/referer, reader view, archive sites and/or add a custom site): Tried clearing cookies, changing user-agent, using reader view, and archiving the site. Poking around the HTML. 

Browser (main version): Firefox 125.0.3

BPC-extension version (number): 3.6.7.1

Apologies for not searching thoroughly with my last issue - this time I've searched all 3 repos issues before opening this one. Thanks for your hard work on this!

**Comment by MarcvsCvrtivs** [comment]
Confirming what ashprice has reported above, I had the same issue with both Firefox and Chrome 124.0.6367.119

**Comment by eylenburg** [comment]
Same here

**Comment by qnoum** [comment]
Works fine by me though.
FF Dev 126.0b9, BPC 3.6.7.1

**Comment by AntonioBergallo** [comment]
First of all, thanks for making this great add-on available.

I had the same problem today with the earlier version. I installed the latest one, but the archive fix does not seem to be working for every case. For some links, it does not load and instead returns "BPC > Try for full article text (no need to report issue for external site): | [archive.today] | [archive.md]" or only a blankpage. Reloading the page seems to do the trick, though. Is any future workaround going to break the paywall like before or is fetching from archive the definitive solution?

**Comment by richtoy** [comment]
I don't think the fix made yesterday is yet included in the xpi file required by Firefox on android.  The fix will therefore only be in the next release.  Is it possible to produce an updated xpi file?

**Comment by bakoven** [comment]
> I don't think the fix made yesterday is yet included in the xpi file required by Firefox on android. The fix will therefore only be in the next release. Is it possible to produce an updated xpi file?

I hope we get a fix soon or as suggested an updated xpi file

**Comment by richtoy** [comment]
> > I don't think the fix made yesterday is yet included in the xpi file required by Firefox on android. The fix will therefore only be in the next release. Is it possible to produce an updated xpi file?
> 
> I hope we get a fix soon or as suggested an updated xpi file

Indeed, is there a reason that chrome.crx was updated to 3.6.8.0 but the Firefox.xpi was not? The workaround using the zip works on the desktop but I still need xpi for Firefox nightly on Android.

**Comment by richtoy** [comment]
> "The workaround using the zip works on the desktop" @richtoy I just use Firefox on my desktop and I have downloaded bypass-paywalls-firefox-clean-master.zip, but can't work out how to update from the file? Any help would be appreciated.

The last paragraph of Installation instructions explains:
"Or load a temporary add-on in regular Firefox (go to about:debugging#/runtime/this-firefox & load manifest.json from unpacked (master-zip) folder)."

**Comment by bakoven** [comment]
> Just wait for the next release (delayed for signing) or switch to Vivaldi + crx-installation. PS RTFM ;-)
I will do that, when do you anticipate the new release to be signed?
